apps/widget-configurator/src/app/embedDialog/utils/vanillaNoDepsExample.ts

```diff
@@ -0,0 +1,26 @@
+import type { CowSwapWidgetParams } from '@cowprotocol/widget-lib'
+
+export function vanillaNoDepsExample(params: CowSwapWidgetParams): string {
+  const paramsSanitized = {
+    ...params,
+    provider: `<eip-1193 provider>`,
+  }
+
+  return `
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>CoWSwap Widget demo</title>
+  <script src="https://cdn.jsdelivr.net/npm/@cowprotocol/widget-lib@latest/index.iife.js"></script>
+</head>
+<body>
+  <div id="app"></div>
+  <script>
+    const params = ${JSON.stringify(paramsSanitized, null, 4)}
+
+    cowSwapWidget.cowSwapWidget(document.getElementById("app"))
```

Comment:
The `params` are defined but not user anywhere
![image](https://github.com/cowprotocol/cowswap/assets/43217/e8b6fa75-b909-4a12-8482-dfa8aa920960)

Reply:
Thanks! Fixed